{"uuid": "2c59030e-7a59-4519-89b5-9e8abe3bf2ce", "children": ["3f0893f4-1551-4327-a5ed-93a2a9a4b240"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/test/Desktop/sprint_7/conftest.py\", line 22, in driver\n    driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\n"}, "start": 1728419169935, "stop": 1728419172585}], "start": 1728419169935, "stop": 1728419172640}